include/deal.II/grid/grid_out.h

```diff
@@ -1080,6 +1080,39 @@ class GridOut
                   std::ostream                      &out) const;
 
   /**
+   * Write triangulation in VTU format for each processor, then combine these
+   * files into a .pvtu file. The output is in the form
```

Comment:
-> "Write triangulation in VTU format for each processor, and add a .pvtu file for visualization in Visit or Paraview that describes the collection of VTU files as all part of the same simulation."